src/annotator/integrations/vitalsource.js

```diff
@@ -12,8 +12,8 @@ import { injectClient } from '../hypothesis-injector';
  * @typedef {import('../../types/annotator').Anchor} Anchor
  * @typedef {import('../../types/annotator').Annotator} Annotator
  * @typedef {import('../../types/annotator').Integration} Integration
- * @typedef {import('../../types/annotator').Selector} Selector
  * @typedef {import('../../types/annotator').SidebarLayout} SidebarLayout
+ * @typedef {import('../../types/api').Selector} Selector
```

Comment:
...just...ordering?

Reply:
The `Selector` type is declared in `types/api.js` but used to be implicitly re-exported by `types/annotator.js`. That implicit re-exporting no longer happens.